lab_7_llm/main.py

```diff
@@ -193,3 +274,11 @@ def run(self) -> dict | None:
         Returns:
             dict | None: A dictionary containing information about the calculated metric
         """
+        data = pd.read_csv(self._data_path).drop(columns=['Unnamed: 0'])
+
+        for metric in self._metrics:
+            if metric.value == "accuracy":
+                accuracy_metric = load(metric.value).compute(references=data['target'],
+                                                             predictions=data['predictions'])
+                return dict(accuracy_metric)
```

Comment:
Isn't the `accuracy_metric` a dictionary anyway?

Reply:
I had problems with Mypy checks, so I've written it such way but now I've changed this function and delete dict() method

done